foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -22,19 +30,20 @@ Once you are done working on your feature branch and are ready to bring the comm
 
 Merges are done by using the command `git merge <branch_name>` which will take the changes you’ve committed in `branch_name` and add them to the branch that you’re currently on. You can see an example of a `develop` branch being created, committed to, and then merged to `main` in the diagram below.
 
-<pre class="mermaid">
----
-title: Example of Git Branching
----
+## <pre class="mermaid">
```

Comment:
Nit: This is used to display a diagram, and needs to be in a very specific format to display correctly.
Can everything between this line and line 46 be reverted back to what it was before please?